base/doc/ltnews41.tex

```diff
@@ -628,7 +776,10 @@ \subsection{Collecting environment bodies verbatim}
   % Some code is shown here
   $y = mx + c$
 \end{verbatim}
-\end{center}
+%% why does this get centred? 
+%%   it looks very bad! 
+%%  AND the example is weird: the "code" is in fact a math formula!!  
```

Comment:
It's verbatim so the fact that it's maths is to show `$` grabbing - centring is just because otherwise you only need `verbatim`.

Reply:
I changed to flushright, which I think makes it a little clearer